doc/ch-grow_desc.rst

```diff
@@ -316,120 +231,35 @@ bug-compatible for the :code:`COPY` instructions.
 
    2. If within a directory named in sources, symlinks are copied as symlinks.
 
-The following :code:`COPY` features are not yet implemented:
-
-* From prior stage by index or name, because multi-stage build is not yet
-  implemented.
-
-* From an arbitrary image by name.
-
-* List form of the instruction.
-
-* Escaping special characters in filenames.
-
-* :code:`COPY` with Dockerfile on standard input (e.g., :code:`ch-grow -f -`)
-  *does* work, because :code:`ch-grow` currently requires a context directory
-  in this case (see “Usage” above).
-
 We expect the following differences to be permanent:
 
 * Wildcards use Python glob semantics, not the Go semantics.
 
 * :code:`COPY --chown` is ignored, because it doesn't make sense in an
   unprivileged build.
 
-:code:`ENTRYPOINT`
```

Comment:
Why was the discussion for `ENTRYPOINT` removed?

Reply:
I moved it to issue #780. It would be available by following the project link above, and once I finish #770, the warning message about being ignored would reference the same issue number.

Reply:
Ah. Gotcha1